builtins/generic.ispc

```diff
@@ -4,5 +4,2292 @@
 
 // This file provides the target-independent definitions for builtin functions.
 
-// Empty for now.
-;
+#define ISPC_INTERNAL_STDLIB_COMPILATION
+#include "core.isph"
+#undef ISPC_INTERNAL_STDLIB_COMPILATION
+
+#define ISPC_INTERNAL_STDLIB_COMPILATION
+#include "builtins.isph"
+#undef ISPC_INTERNAL_STDLIB_COMPILATION
+
+#if TARGET_WIDTH == 2
+#define DOUBLE_WIDTH 4
+#elif TARGET_WIDTH == 4
+#define DOUBLE_WIDTH 8
+#elif TARGET_WIDTH == 8
+#define DOUBLE_WIDTH 16
+#elif TARGET_WIDTH == 16
+#define DOUBLE_WIDTH 32
+#elif TARGET_WIDTH == 32
+#define DOUBLE_WIDTH 64
+#elif TARGET_WIDTH == 64
+#define DOUBLE_WIDTH 128
+#else
+#error Unknown value of TARGET_WIDTH
+#endif
+
+// Macros for attributes
+#define NOESCAPE __attribute__((noescape))
```

Comment:
I think it worth going to long prefixed names for macros in stdlib. As in case of name collision we redefine it and undef in the end of the file.

I think something like `ISPC_STDLIB_NOESCAPE` should be better. Same for other macros, which are implementation detail.

Reply:
These macro definitions are internal to this file. The undef at the end emphasizes this, and it is actually unnecessary unless someone decides to include stdlib.ispc somewhere. Because of this, there is no point in adding extra prefixes that would create visual noise across the file while being virtually useless at the same time.

Reply:
@nurmukhametov, as we discussed it would be good to change macro names in *.isph files using longer prefixes. Do you want to do it as part of this PR or create another one? If you want to do in a separate one, feel free to merge this one.